src/trix/views/attachment_view.coffee

```diff
@@ -109,8 +109,12 @@ class Trix.AttachmentView extends Trix.ObjectView
   # Attachment delegate
 
   attachmentDidChangeUploadProgress: ->
-    value = @attachment.getUploadProgress()
-    @findProgressElement()?.value = value
+    if element = @findProgressElement()
+      value = @attachment.getUploadProgress()
+      if value < 100
+        element.value = value
+      else
+        element.style.display = "none"
```

Comment:
Without this change, the progress bar will remain at 100% if you're editing an attachment when it finishes uploading. Would be better to receive a delegate call informing the view that its attachment is no longer pending (it has a `url` attribute). Currently it's only informed about progress: https://github.com/basecamp/trix/blob/d4b81b80e23c1a1ef08486f1ec9982808ec66fa6/src/trix/views/attachment_view.coffee#L7-L10